napari/__main__.py

```diff
@@ -145,6 +145,17 @@ def _run():
         default=0,
         help="increase output verbosity",
     )
+    parser.add_argument(
+        '-w',
+        '--widget',
```

Comment:
I wonder if this could be more general than `widget`, say `with`? Your opening napari `with` a plugin? maybe that's too overloaded? maybe it's not clear how we'll use it otherwise, but curious what @jni thinks? I love the functionality

Reply:
that's funny... I think I came to `-w` thinking about `--with` in my head... I'd be happy to expand the scope of this if everyone wants to.

Reply:
Yep, love `--with`. Like, really love it. =)

Reply:
changed.